src/index.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable import/extensions */
+/* eslint-disable import/no-cycle */
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable no-use-before-define */
+/* eslint-disable max-classes-per-file */
+import './styles/main.scss';
+import saveScore from './modules/save.js';
+import { renderScores } from './modules/render.js';
```

Comment:
- [ ] Kindly make sure you fix as many as possible all the linters errors and do not disable them in all the `.js` files as it is required for no custom changes to the linters and for best practices. 
**Make sure you apply this approach in all your files**

Reply:
ok some of those disables i dont need

Reply:
Hi @david63011 

If you don't need them, it would be good to remove them from your codebase

Thanks

Reply:
ok I will !